src/Program.cs

```diff
@@ -87,26 +73,9 @@ private static async Task RunAsync()
             catch (Exception ex)
             {
                 Console.ForegroundColor = ConsoleColor.Red;
-
-                var innerException = ex.InnerException;
-                if (innerException != null)
-                {
-                    while (innerException != null)
-                    {
-                        Console.WriteLine(innerException.Message);
-                        innerException = innerException.InnerException;
-                    }
-                }
-                else
-                {
-                    Console.WriteLine(ex.Message);
-                }
-            }
-            finally
-            {
+                Console.WriteLine($"An error occurred: {ex}");
```

Comment:
Recommended pattern for exception logging it to just call "ToString()" on the exception. This will do the right thing - print the message, stack trace and inner exceptions. Also .net developers are pretty accustomed to reading this format.